app/views/shared/authentication/_forem_creator_signup.html.erb

```diff
@@ -40,65 +40,67 @@
       <% end %>
 
       <div class="align-center">
-        <p class="pb-4 fs-2xl m:fs-3xl lh-tight fw-bold">Let's start your Forem journey!</p>
-        <p class="color-base-70 fs-l m:fs-xl">Create your admin account first.</p>
-        <p class="color-base-70 fs-l m:fs-xl">Then we'll walk you through your Forem setup.</p>
+        <p class="pb-4 fs-2xl m:fs-3xl lh-tight fw-bold"><%= t("views.auth.creator.text1") %></p>
+        <p class="color-base-70 fs-l m:fs-xl"><%= t("views.auth.creator.text2") %></p>
+        <p class="color-base-70 fs-l m:fs-xl"><%= t("views.auth.creator.text3") %></p>
       </div>
 
       <div class="crayons-field mt-6">
-        <%= f.label :name, "Name", for: "name", class: "crayons-field__label" %>
+        <%= f.label :name, t("views.auth.register.field.name.label"), for: "name", class: "crayons-field__label" %>
         <%= f.text_field :name, id: "name", placeholder: "John Doe", class: "crayons-textfield js-creator-signup-name", required: true %>
 
         <div class="js-creator-signup-username-hint-row hidden">
           <div class="fs-xs color-base-60 flex">
-            <span>Your username will be&nbsp;</span>
-            <button type="button" aria-label="Edit username" class="cursor-pointer border-none bg-transparent p-0 js-creator-edit-username flex color-base-90" role="button">
-              <span class="js-creator-signup-username-hint"></span>
-              <%= inline_svg_tag("pencil.svg", class: "cursor-pointer crayons-icon opacity-75 ml-1", aria_hidden: true, title: "Edit username") %>
-            </button>
+            <%= t("views.auth.creator.hint.text_html", start: tag("span", nil, true), end: "</span>".html_safe, # rubocop:disable Rails/ContentTag
+                                                       button: tag.button(
```

Comment:
@yheuhtozr I looked into the Travis failures and it seem like this change has caused the error.
<img width="567" alt="Screenshot 2021-11-30 at 12 21 23" src="https://user-images.githubusercontent.com/2786819/144029470-c9e05163-8c47-468e-bf50-a1aea4aae8f4.png">


As you can see from the screenshot above it is not finding the class `js-creator-signup-username-hint`.

When I look at the DOM I see the following, hence as you can see the span is not rendering correctly. 

<img width="567" alt="Screenshot 2021-11-30 at 12 24 01" src="https://user-images.githubusercontent.com/2786819/144030129-b51af695-ec3a-41a5-92a0-171284e723fa.png">
 

To test this feature on the frontend you can do the following: 

You can either drop your entire database via `rails:db drop`
and enable the feature flag `FeatureFlag.enable(:creator_onboarding)`

or

You can open a rails console via `rails c` and do the following:
```
# Enable the feature flag
FeatureFlag.enable(:creator_onboarding)

# Delete all users 
User.destroy_all

# Set `waiting_on_first_user` to be true
Settings::General.waiting_on_first_user = true

# If you've logged in before set the mascot id to nil
Settings::General.mascot_user_id = nil
```

Now, run your server and navigate to http://localhost:3000/ and you will see the "Forem Creator Setup" form.

Reply:
Thank you for your time and the detailed guide! It really helped me doing it right. So this is the page showing only once when DB is empty, that's why I didn't know what it was.

Now the spec is green, and as a by-effect, I realized a couple of useless `rubocop:disable Rails/ContentTag` comments (originally meant to cope with [a bug existed in some versions of Rubocop](https://github.com/rubocop/rubocop-rails/issues/556)) still left in some files I contributed. They are supposed to do no harm but you can remove it whenever convenient.